src/contribute/intellij.md

```diff
@@ -15,6 +15,22 @@ section_menu: section-menu/contribute.html
 >
 > Mastering IntelliJ is, ultimately, your responsibility.
 
+## Create a working Gradle module
+
+1. Create an empty IntelliJ project outside of the Beam source tree.
+2. Under Project Structure > Project, select a Project SDK.
+3. Under Project Structure > Modules, click the + sign to add a module and select "Import Module".
+    1. Select the directory containing the Beam source tree.
+    2. Tick the "Import module from external model" button and select Gradle from the list.
+    3. Tick the following boxes.
+       * Use auto-import
+       * Create separate module per source set
+       * Store generated project files externally
+       * Use default gradle wrapper
+
+This should result in a working Gradle project. Build the project by executing the "build" task in
+the root Gradle module.
+
 ## Enable Annotation Processing
```

Comment:
I don't believe we need this whole section.

Reply:
By this whole section do you mean the "Enable Annotation Processing" section?

Reply:
Yes, `target` folders refer to Maven.

Also, try out the quickstart manually using intellij in a new project.

Reply:
Ah. So does that mean there's no gradle version of those directories then if the solution is to remove the entire section?

Also yes, I'll do that. I already did so for the gradle setup directions, but I ignored the rest of the instructions (annotations, checkstyle, and codestyle).

Reply:
Based upon your current set of instructions for intellij, I don't believe we need anything additional for the annotation processing.